packages/google-compute-engine-oslogin/google_oslogin_control

```diff
@@ -154,6 +154,7 @@ modify_pam_config() (
     pam_account_oslogin="account    optional pam_oslogin_admin.so"
     pam_account_admin="account    requisite pam_oslogin_login.so"
     pam_session_homedir="session    optional pam_mkhomedir.so"
+    pam_account_su="account    optional pam_oslogin_login.so"
```

Comment:
in linux we have mapped success to 'bad', so that oslogin users will be *prevented* from su. this line won't have the same effect